src/SFA.DAS.EmployerFeedback.Application.UnitTests/Commands/UpdateFeedbackTransaction/UpdateFeedbackTransactionCommandTests.cs

```diff
@@ -0,0 +1,55 @@
+using NUnit.Framework;
+using SFA.DAS.EmployerFeedback.Application.Commands.UpdateFeedbackTransaction;
+using SFA.DAS.EmployerFeedback.Application.Models;
+using System;
+
+namespace SFA.DAS.EmployerFeedback.Application.UnitTests.Commands.UpdateFeedbackTransaction
+{
+    [TestFixture]
+    public class UpdateFeedbackTransactionCommandTests
+    {
+        [Test]
+        public void Properties_SetAndGet_ShouldReturnExpectedValues()
+        {
+            var id = 123L;
+            var templateId = Guid.NewGuid();
+            var sentCount = 5;
+            var sentDate = DateTime.UtcNow;
+
+            var command = new UpdateFeedbackTransactionCommand
+            {
+                Id = id,
+                TemplateId = templateId,
+                SentCount = sentCount,
+                SentDate = sentDate
+            };
+
+            Assert.That(command.Id, Is.EqualTo(id));
```

Comment:
This has different styles of assertions in different test classes - we need a consistent style in the whole api - which should be fluent assertions ones

Reply:
Done